markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -52,36 +52,97 @@ void MarkhorHWInterface::setupCTREDrive()
+    rear_left_drive->SelectProfileSlot(0, 0);
+    rear_left_drive->Config_kF(0, 0, timeout_ms_);
+    rear_left_drive->Config_kP(0, 0.3, timeout_ms_);
+    rear_left_drive->Config_kI(0, 0.005, timeout_ms_);
+    rear_left_drive->Config_kD(0, 0, timeout_ms_);
   }
   if (nh.getParam("/markhor/markhor_tracks_node/front_right", drive_fr_id) == true)
   {
     front_right_drive = std::make_unique<TalonSRX>(drive_fr_id);
-    front_right_drive->SetNeutralMode(NeutralMode::Brake);
+    front_right_drive->SetNeutralMode(NeutralMode::Coast);
     front_right_drive->SetInverted(true);
+    front_right_drive->ConfigFactoryDefault();
+    front_right_drive->ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative , 0, timeout_ms_);
+    front_right_drive->SetSensorPhase(true);
+    front_right_drive->ConfigSupplyCurrentLimit(current_limit_config);
+    front_right_drive->ConfigNominalOutputForward(0, timeout_ms_);
+    front_right_drive->ConfigNominalOutputReverse(0, timeout_ms_);
+    front_right_drive->ConfigAllowableClosedloopError(0, 0, timeout_ms_);
+    front_right_drive->SelectProfileSlot(0, 0);
+    front_right_drive->Config_kF(0, 0, timeout_ms_);
+    front_right_drive->Config_kP(0, 0.3, timeout_ms_);
+    front_right_drive->Config_kI(0, 0.005, timeout_ms_);
+    front_right_drive->Config_kD(0, 0, timeout_ms_);
   }
   if (nh.getParam("/markhor/markhor_tracks_node/rear_right", drive_rr_id) == true)
   {
     rear_right_drive = std::make_unique<TalonSRX>(drive_rr_id);
-    rear_right_drive->SetNeutralMode(NeutralMode::Brake);
+    rear_right_drive->SetNeutralMode(NeutralMode::Coast);
+    rear_right_drive->ConfigFactoryDefault();
+    rear_right_drive->ConfigSelectedFeedbackSensor(FeedbackDevice::CTRE_MagEncoder_Relative , 0, timeout_ms_);
+    rear_right_drive->SetSensorPhase(false);
+    rear_right_drive->ConfigSupplyCurrentLimit(current_limit_config);
+    rear_right_drive->ConfigNominalOutputForward(0, timeout_ms_);
+    rear_right_drive->ConfigNominalOutputReverse(0, timeout_ms_);
+    rear_right_drive->ConfigAllowableClosedloopError(0, 0, timeout_ms_);
+
+    rear_right_drive->SelectProfileSlot(0, 0);
+    rear_right_drive->Config_kF(0, 0, timeout_ms_);
+    rear_right_drive->Config_kP(0, 0.1, timeout_ms_);//TODO : Set this value to 0.3 when encoder is fixed
+    rear_right_drive->Config_kI(0, 0.002, timeout_ms_);//TODO : Set this value to 0.005 when encoder is fixed
+    rear_right_drive->Config_kD(0, 0, timeout_ms_);
+
+    front_left_drive->Set(ControlMode::Velocity, 0);
+    front_right_drive->Set(ControlMode::Velocity, 0);
+    rear_left_drive->Set(ControlMode::Velocity, 0);
+    rear_right_drive->Set(ControlMode::Velocity, 0);
+
   }
 }
 
 void MarkhorHWInterface::write()
 {
-  double diff_ang_speed_front_left = cmd[0];
-  double diff_ang_speed_front_right = cmd[1];
-  double diff_ang_speed_rear_left = cmd[2];
-  double diff_ang_speed_rear_right = cmd[3];
+  double diff_ang_speed_front_left = cmd[0] * 125;
```

Comment:
Ici aussi au moins une variable a serait mieux.

Reply:
Will be fixed avec un scaling de l'encodeur (configselectedfeedbackcoefficient) same avec ki kp

Reply:
Mais pour les autres valeurs oui sa prends un scaling calculé avec le nombre de steps per turn pour que le input en radians/s represente ce qu'on a sur les tracks

Reply:
D'accord, ouin tu n'as pas le choix de mettre ça dans le code. C'est comme mon multiplicator dans les flippers.

Reply:
ouais, faudrait faire le calcul correct pour avoir une valeur qui correspond au nombre de rad/seconde des flippers

Reply:
Ok, est-ce que tu veux faire ça dans cette PR ou bien tu voudrais le mettre en issue ?

Reply:
j'penses que ça vas rester en issue, mais pour le moment on mergera pas la PR tout de suite, y'a quelques issues avec le controlleur en virage et lorsqu'une track est bloquée, c'est fort probablement du au windup de l'intégrale, je vais essayer de le limiter et enable des features sur l'intégrale cette semaine

Reply:
Ok, d'abord est-ce que tu veux créer l'issue tout de suite, c'est possible avec l'interface de Github. 

![image](https://user-images.githubusercontent.com/12837376/141692794-466768a5-3489-4cf1-8701-8b43d4cc671b.png)


Aussi d'abord est-ce que tu veux qu'on close là PR pour l'instant considérant qu'il a pas mal de truc qui reste à faire/ testé ?